easybuild/easyconfigs/u/UDA/UDA-2.9.3-GCC-13.2.0.eb

```
@@ -0,0 +1,63 @@
+easyblock = 'CMakeNinja'
+
+name = 'UDA'
+version = '2.9.3'
+
+homepage = 'https://ukaea.github.io/UDA/'
+description = """Universal Data Access"""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ukaea/UDA/archive/refs/tags']
+sources = ['%(version)s.zip']
+checksums = ['30c6e6d5cbdf2958df499f60a46ea43b1d1e44ae3c31477c9702bc8ce62b7f5b']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('Ninja', '1.11.1'),
+    ('libMemcached', '1.0.18'),
+]
+
+dependencies = [
+    ('libxml2', '2.11.5'),
+    ('libtirpc', '1.3.4'),
+    ('Boost', '1.83.0'),
+    ('fmt', '10.2.1'),
+    ('spdlog', '1.12.0'),
+    ('CapnProto', '1.0.1.1'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+]
+max_parallel = 1
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
+configopts += '-DBOOST_ROOT=${EBROOTBOOST} '
+configopts += '-DENABLE_CAPNP=ON '
+configopts += '-DSSLAUTHENTICATION=ON '
```

```suggestion
configopts = ' '.join([
    '-DENABLE_CAPNP=ON',
    '-DSSLAUTHENTICATION=ON',
])
```

The first 2 options are already dealt with by the EBlock, also switching to this sintax to make it harder to make `+=` or trailing white-space related mistakes